canonical-web-and-design/charmhub.io
# Add charms and bundles listing pages

## Done

- Add charms and bundles listing pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045/<charm_name>/listing
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to http://localhost:8045/<charm_name>/listing and compare to [charm design](https://app.zeplin.io/project/5d66544639bbe3139a99e168/screen/5f06ee9db569f66c57e029fe)
- Go to http://localhost:8045/bundle/listing and compare to [bundle design](https://app.zeplin.io/project/5d66544639bbe3139a99e168/screen/5d6656f1641a109bd392baea)


## Issue / Card

Fixes #143
Fixes #144
